src/BasketBundle/Block/BasketBlockService.php

```diff
@@ -22,7 +22,7 @@
 /**
  * @author Xavier Coureau <xcoureau@ekino.com>
  */
-class BasketBlockService extends BaseBlockService
+class BasketBlockService extends AbstractBlockService
```

Comment:
@greg0ire do you know the answer? It should be AbstractBlockService or AbstractAdminBlockService?